app/src/main/java/org/coepi/android/ui/cen/CENFragment.kt

```diff
@@ -37,4 +41,17 @@ class CENFragment : Fragment() {
             textMyCurrentCEN.text = it
         }
     }.root
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        var curcen = viewModel.curcen.toString()
+
+        textCENReport.setText(curcen.toCharArray(),0,curcen.length);
+        postReport.setOnClickListener(){
+            val cen_from_other = textCENReport.text;
+            var cenDao: RealmCenDao = RealmCenDao(RealmProvider(view.context));
```

Comment:
This should be injected in the view model. Never use Dao / Api etc directly from UI (views, fragments)

Reply:
mv DAO  to VM using repo already passes ad paramenter